internal/pkg/mount/system.go

```diff
@@ -190,8 +190,12 @@ func SystemPartitionMount(ctx context.Context, r runtime.Runtime, logger *log.Lo
 	}
 
 	systemInformation, err := r.GetSystemInformation(ctx)
-	if err != nil && !errors.Is(err, context.Canceled) && state.IsNotFoundError(err) {
-		return err
+	if err != nil {
```

Comment:
I don't think this has anything to do with timeouts, as it fails after installing Talos, which is way late in the boot sequence

Reply:
yes, I found the actual bug.